CHANGELOG.md

```diff
@@ -1569,7 +1569,7 @@ the Datadog user will need to be modified from ```'datadog'@'localhost'``` to ``
 * If you were using a custom check that needed python dependencies you will have to reinstall them using the bundled pip:
 
        ```
-sudo /opt/datadog-agent/embedded/bin/pip install YOUR_DEPENDENCY
+sudo /opt/stackstate-agent/embedded/bin/pip install YOUR_DEPENDENCY
        ```
 * Configuring checks in datadog.conf for checks.d is deprecated and won't work anymore. Please configure your checks by editing the yaml files in the conf.d directory.
```

Comment:
@craffit suggests CHANGELOG doesn't make it to the package - I'll remove this change